Data sync to custom Salesforce objects (#811)
rudderlabs/rudderstack-docs: docs/sources/reverse-etl/features/visual-data-mapper.mdx

1. Click on the **Map another field** option. 
2. Type the name of the custom destination field that you want to create, as shown:

<img src="../../../assets/warehouse-actions-sources/vdm-7.png" alt="Map another field" />

3. Finally, click on **Create**.

<div class="warningBlock">
This feature is currently available for the following  destinations:
<ul>
 <li><a href="https://rudderstack.com/docs/destinations/analytics/amplitude/">Amplitude</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/braze/">Braze</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/customer.io/">Customer.io</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/analytics/google-analytics-ga/">Google Analytics</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/iterable/">Iterable</a></li>
  <li><a href="https://rudderstack.com/docs/destinations/marketing/klaviyo/">Klaviyo</a></li>
</ul>
</div>

Once you have mapped all the source columns to the destination fields, click on **Next** to complete the destination configuration.

### Syncing data to custom Salesforce objects using VDM

> RudderStack **does not support** creating new custom <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> objects using the above-mentioned steps. However, you can still use the Visual Data Mapping feature sync your data to these objects by following the below steps:

Reviewer: the above mentioned steps are about creating custom fields not objects. We do not allow creation of custom fields in salesforce.

Reviewer: for none custom objects can be created.

Author: thanks @ruchiramoitra I've updated the section accordingly.

> RudderStack **does not support** creating new custom <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> objects using the above-mentioned steps. However, you can still use the Visual Data Mapping feature sync your data to these objects by following the below steps:

Author: ```suggestion
RudderStack **does not support** creating new custom fields for the <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> destination using the above-mentioned steps. 

However, you can still use the Visual Data Mapping feature sync your data to the custom Salesforce objects by following the below steps:
```

1. Click on the **Map another field** option. 
2. Type the name of the custom destination field that you want to create, as shown:

<img src="../../../assets/warehouse-actions-sources/vdm-7.png" alt="Map another field" />

3. Finally, click on **Create**.

<div class="warningBlock">
This feature is currently available for the following  destinations:
<ul>
 <li><a href="https://rudderstack.com/docs/destinations/analytics/amplitude/">Amplitude</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/braze/">Braze</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/customer.io/">Customer.io</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/analytics/google-analytics-ga/">Google Analytics</a></li>
 <li><a href="https://rudderstack.com/docs/destinations/marketing/iterable/">Iterable</a></li>
  <li><a href="https://rudderstack.com/docs/destinations/marketing/klaviyo/">Klaviyo</a></li>
</ul>
</div>

Once you have mapped all the source columns to the destination fields, click on **Next** to complete the destination configuration.

> ### Syncing data to custom Salesforce objects using VDM

Reviewer 2: ```suggestion
#### Syncing data to custom Salesforce objects
```

RudderStack **does not support** creating new custom fields for the <Link to="/destinations/streaming-destinations/salesforce/">Salesforce</Link> destination using the above-mentioned steps. 

> However, you can still use the Visual Data Mapping feature sync your data to the custom Salesforce objects by following the below steps:

Reviewer 2: ```suggestion
However, you can still use the Visual Data Mapping feature to sync your data to the custom Salesforce objects by following the below steps:
```
